### PR TITLE
Dark mode & responsiveness

### PR DESCRIPTION
**Pull request by amartya-rs**
In this PR:
- added dark mode
- added responsiveness

Files to check: [review me](https://github.com/amartya-rs/hhQuiz/pull/4/files/1d3483d049404c61c970ce8e696e2ed53a954527..70f41a33c5ec72c44e579c371a629f67e47beeb0)

Preview: https://deploy-preview-4--hh-quiz.netlify.app/

### CODE REVIEW

File: src/pages/quiz-page/QuizPage.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { useData } from "../../context/data-context";
 import { LeftArrow, RightArrow } from "../../assets/icons";
 import { useNavigate } from "react-router-dom";
 import { useEffect } from "react";
+import { useTheme } from "../../context/theme-context";
```
Comment (iRohitGaur) [inline src/pages/quiz-page/QuizPage.jsx:5]: suggestion: you have an index file in context folder. better to export from there to get a clean import here

**Review by iRohitGaur — APPROVED** [review]
Nice work!

Gave some suggestions, please look into it.

Good to merge with updates 👍🏻

**Review by vidhi-thakur — APPROVED** [review]
code looks fine, it's nice that you have added comments for future reference 👍

File: src/pages/quiz-page/QuizPage.jsx (hunk 1 above)
Comment (amartya-rs) [reply src/pages/quiz-page/QuizPage.jsx:5]: I have only two context, so I didn't create any index file for exporting.

**Pull request merged**